src/Crypt.php

```diff
@@ -45,10 +45,10 @@ class Crypt
 	public function __construct(CipherInterface $cipher = null, Key $key = null)
 	{
 		// Set the encryption cipher.
-		$this->cipher = isset($cipher) ? $cipher : new Crypto;
+		$this->cipher = $cipher !== null ? $cipher : new Crypto;
```

Comment:
Would elvis operators be better here and for the key?

``php
$this->cipher = $cipher ?: new Crypto;
```